ods_ci/tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -141,7 +141,7 @@ Delete Tabname Instance For Installed Operator
     Wait Until Element Is Enabled   //button[contains(text(),"Delete")]
     Click Button    Delete
 
-Check If Operator Is Already Installed In Opneshift
+Check If Operator Is Already Installed In Openshift
```

Comment:
Would it be possible to have all these refactoring changes in a separate PR, please?

Reply:
ok @jstourac